File: CS/PrintingExample/ReceptionDeskDBInitializer.cs
using System.Collections.Generic;
using System.Data.Entity;

namespace PrintingExample {
    public class ReceptionDeskDBInitializer : CreateDatabaseIfNotExists<ReceptionDeskContext> {
        protected override void Seed(ReceptionDeskContext context) {
            base.Seed(context);
            List<Doctor> Doctors = new List<Doctor>() {
            new Doctor(1,"Stomatologist"),
            new Doctor(2, "Ophthalmologist"),
            new Doctor(3, "Surgeon"),
            };
            Doctors.ForEach(x => context.Doctors.Add(x));
            context.SaveChanges();
        }
    }
}

File: VB/PrintingExample/ReceptionDeskDBInitializer.vb
Imports System.Collections.Generic
Imports System.Data.Entity

Namespace PrintingExample

    Public Class ReceptionDeskDBInitializer
        Inherits CreateDatabaseIfNotExists(Of ReceptionDeskContext)

        Protected Overrides Sub Seed(ByVal context As ReceptionDeskContext)
            MyBase.Seed(context)
            Dim Doctors As List(Of Doctor) = New List(Of Doctor)() From {New Doctor(1, "Stomatologist"), New Doctor(2, "Ophthalmologist"), New Doctor(3, "Surgeon")}
            Doctors.ForEach(Sub(x) context.Doctors.Add(x))
            context.SaveChanges()
        End Sub
    End Class
End Namespace
